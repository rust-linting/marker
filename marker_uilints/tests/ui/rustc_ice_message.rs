//@rustc-env:RUST_BACKTRACE=0
//@normalize-stderr-test: "rustc 1\..* running on .*" -> "rustc <version> running on <target>"
//@normalize-stderr-test: "(?ms)query stack during panic:\n.*end of query stack\n" -> ""
//@normalize-stderr-test: "marker_rustc_driver .*" -> "marker_rustc_driver bar"
//@normalize-stderr-test: "item.rs:.*" -> "item.rs"

// This function will trigger a panic in rustc's driver.
// I want to see how the ICE message looks and make sure that it
// links to the rust-marker/marker repo.
fn rustc_driver_please_ice_on_this() {}

fn main() {}
