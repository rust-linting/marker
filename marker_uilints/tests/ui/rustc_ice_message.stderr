thread 'rustc' panicked at marker_rustc_driver/src/conversion/marker/item.rs
this is your captain talking, we are about to ICE
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

error: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-marker/marker/issues/new?template=panic.yml

note: rustc <version> running on <target>

note: marker_rustc_driver bar

note: Achievement Unlocked: [Free Ice Cream]

